open-metadata-resources/open-metadata-samples/access-services-samples/subject-area-client-samples/README.md

```diff
@@ -0,0 +1,15 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Subject Area Open Metadata Access Service (OMAS) Client Samples
+
+These samples shows how to call the Subject Area OMAS's client.
+There is a sample for each of the Subject Area OMAS's interfaces.
+
+* **[Subject Area Definitions](Subject-Area-Definitions-creation.md)** shows how to create Subject Area Definitions.
```

Comment:
I am not sure this link will work. It is saying that there is a file called Subject-Area-Definitions-creation.md in the current folder. I think the brackets will need to be (subject-area-definition-sample) so the link resolves to the README.md at that location.

Reply:
See follow up commit for changes